smpe/bld/service/ptf-bucket.txt

```diff
@@ -5,5 +5,5 @@
 #----------------------------------------------------------------------#
 #
 # format: <ptf1> <ptf2> - <apar1> <apar2> [<sec/int apar>]
-#UO02041 UO02042 - IO28859 IO28860   IO28865 - Mon Jun 13 18:46:28 UTC 2022
+UO02041 UO02042 - IO28859 IO28860   IO28865
```

Comment:
should we put this correct line here:
`UO02047 UO02048 - IO28859 IO28860   IO28865`
Since UO02047 and 2048 are invalid anyways. I don't know if automation will introduce conflict when changing this file later?